File: samples/server/petstore/java-vertx-web/src/main/java/org/openapitools/vertxweb/server/model/Dummy200ResponseOuterProp.java
package org.openapitools.vertxweb.server.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import org.openapitools.vertxweb.server.model.Dummy200ResponseOuterPropInnerProp;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class Dummy200ResponseOuterProp   {
  
  private Dummy200ResponseOuterPropInnerProp innerProp;

  public Dummy200ResponseOuterProp () {

  }

  public Dummy200ResponseOuterProp (Dummy200ResponseOuterPropInnerProp innerProp) {
    this.innerProp = innerProp;
  }

    
  @JsonProperty("innerProp")
  public Dummy200ResponseOuterPropInnerProp getInnerProp() {
    return innerProp;
  }
  public void setInnerProp(Dummy200ResponseOuterPropInnerProp innerProp) {
    this.innerProp = innerProp;
  }


  @Override
  public boolean equals(Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    Dummy200ResponseOuterProp dummy200ResponseOuterProp = (Dummy200ResponseOuterProp) o;
    return Objects.equals(innerProp, dummy200ResponseOuterProp.innerProp);
  }

  @Override
  public int hashCode() {
    return Objects.hash(innerProp);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class Dummy200ResponseOuterProp {\n");
    
    sb.append("    innerProp: ").append(toIndentedString(innerProp)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: samples/server/petstore/kotlin-springboot-3/src/main/kotlin/org/openapitools/model/Dummy200ResponseOuterProp.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.Dummy200ResponseOuterPropInnerProp
import jakarta.validation.constraints.DecimalMax
import jakarta.validation.constraints.DecimalMin
import jakarta.validation.constraints.Email
import jakarta.validation.constraints.Max
import jakarta.validation.constraints.Min
import jakarta.validation.constraints.NotNull
import jakarta.validation.constraints.Pattern
import jakarta.validation.constraints.Size
import jakarta.validation.Valid

/**
 * 
 * @param innerProp 
 */
data class Dummy200ResponseOuterProp(

    @field:Valid
    @get:JsonProperty("innerProp") val innerProp: Dummy200ResponseOuterPropInnerProp? = null
) {

}

